doc/source/running.rst

```diff
@@ -300,31 +218,31 @@ Listing all completed backups
 
 .. code-block:: bash
 
-   $ pbmctl --server-address=127.0.0.1:10001 list backups
+   $ pbm list
 
 .. admonition:: Sample output:
 
    .. code-block:: bash
 
-      Metadata file name             -         Description
-      ------------------------------ - --------------------------------------------
-      2018-12-18T19:04:14Z.json      - Test backup 01
+      Backup History:
+
+      2019-09-10T19:04:14Z
 
 .. _pbm.running.backup.restoring: 
 
 Restoring a Backup
 --------------------------------------------------------------------------------
 
-To restore a backup that you have made using ``pbmctl run backup`` you should
-use the ``pbmctl run restore`` command. This command takes two required
-parameters: the storage name and the metadata file. 
+To restore a backup that you have made using ``pbm backup`` you should use the
+``pbm run restore`` command supplying the name of the backup that you intend to
+restore.
 
 .. code-block:: bash
 
-   $ pbmctl run restore --storage=localfs datestamp.json
+   $ pbm run restore 2019-09-10T19:04:14Z
```

Comment:
The command is just `pbm restore` and it needs `--mongodb-uri`.
Here's an example: `./pbm restore 2019-09-16T20:24:06Z --mongodb-uri="mongodb://localhost:27017/"`